### PR TITLE
feat: add Konamimojisplosion package

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
### Overview

Adds a `@codecademy/konamimojisplosion` package exporting a React component that hooks up the [Konami Code](https://en.wikipedia.org/wiki/Konami_Code) to trigger [emojisplosion](https://github.com/JoshuaKGoldberg/emojisplosion). 

We already have this on https://codecademy.com/about/careers. This package will make it easier to put it everywhere.

### PR Checklist

- ~[ ] Related to designs:~
- [x] Related to JIRA ticket: WEB-1356
- [x] I have run this code to verify it works
- ~[ ] This PR includes unit tests for the code change~ _(lots of external dependencies, not a lot of logic)_

### CODE REVIEW

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://605c89ab392c561bd05402fc--gamut-preview.netlify.app>https://605c89ab392c561bd05402fc--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/605c89ab392c561bd05402fc>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/konamimojisplosion@0.2.0-alpha.22290b.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://605c8d4abb9bf31c58363345--gamut-preview.netlify.app>https://605c8d4abb9bf31c58363345--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/605c8d4abb9bf31c58363345>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.2.35-alpha.83e00c.0<br/>@codecademy/gamut-labs@13.1.5-alpha.83e00c.0<br/>@codecademy/gamut@29.4.3-alpha.83e00c.0<br/>@codecademy/konamimojisplosion@0.1.1-alpha.83e00c.0<br/>@codecademy/styleguide@33.3.3-alpha.83e00c.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://605c96b47204d925fd8b4420--gamut-preview.netlify.app>https://605c96b47204d925fd8b4420--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/605c96b47204d925fd8b4420>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.2.35-alpha.e5242d.0<br/>@codecademy/gamut-labs@13.1.5-alpha.e5242d.0<br/>@codecademy/gamut@29.4.3-alpha.e5242d.0<br/>@codecademy/konamimojisplosion@0.1.1-alpha.e5242d.0<br/>@codecademy/styleguide@33.3.3-alpha.e5242d.0<br/></pre>

File: packages/konamimojisplosion/babel.config.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  presets: ['codecademy', '@babel/preset-typescript'],
+  plugins: ['react-anonymous-display-name'],
```
Comment (codecaaron) [inline packages/konamimojisplosion/babel.config.js:3]: I think we just removed this plugin this week.

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://605c9919bb9bf3283e3634eb--gamut-preview.netlify.app>https://605c9919bb9bf3283e3634eb--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/605c9919bb9bf3283e3634eb>Deploy Logs</a></p>

File: packages/gamut/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@codecademy/gamut",
   "description": "Styleguide & Component library for Codecademy",
-  "version": "29.4.1",
+  "version": "29.4.2",
```
Comment (codecaaron) [inline packages/gamut/package.json:4]: Guessing the test-hooks stuff caused some deps to be bumped here?

**Review by codecaaron — APPROVED** [review]
I think we can toss the plugin.  but LGTM

File: packages/konamimojisplosion/babel.config.js (hunk 1 above)
Comment (JoshuaKGoldberg) [reply packages/konamimojisplosion/babel.config.js:3]: Well we didn't remove it everywhere!

Comment (JoshuaKGoldberg) [reply packages/konamimojisplosion/babel.config.js:3]: #1563

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/konamimojisplosion@0.1.1-alpha.3a85d5.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://605c9d845a958a33e0ad28dd--gamut-preview.netlify.app>https://605c9d845a958a33e0ad28dd--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/605c9d845a958a33e0ad28dd>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/konamimojisplosion@0.1.1-alpha.8c70bc.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://605ca3453946a247125c2dd4--gamut-preview.netlify.app>https://605ca3453946a247125c2dd4--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/605ca3453946a247125c2dd4>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/konamimojisplosion@0.1.1-alpha.dac2db.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://605ca9c9ed9ffd4c7866200e--gamut-preview.netlify.app>https://605ca9c9ed9ffd4c7866200e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/605ca9c9ed9ffd4c7866200e>Deploy Logs</a></p>

**Pull request merged**